src/composables/pools/usePoolCreation.ts

```diff
@@ -115,9 +123,11 @@ export default function usePoolCreation() {
   const scaledLiquidity = computed(
     (): Record<string, OptimisedLiquidity> => {
       const scaledLiquidity = {};
-      const modifiedToken = findSeedTokenByAddress(
-        poolCreationState.manuallySetToken
-      );
+      const manuallySetToken =
```

Comment:
I tried always passing the wrapped token address to `poolCreationState.manuallySetToken` instead of doing this, but this computed value stopped updating and returned undefined instead and I couldn't figure out why.